includes/admin/class-wcs-admin-system-status.php

```diff
@@ -261,7 +260,7 @@ private static function set_woocommerce_account_data( &$debug_data ) {
 
 		foreach ( $woocommerce_account_subscriptions as $subscription ) {
 			if ( isset( $subscription['product_id'] ) && self::WCS_PRODUCT_ID === $subscription['product_id'] ) {
-				$has_active_product_key = in_array( $site_id, $subscription['connections'] );
+				$has_active_product_key = in_array( $site_id, $subscription['connections'], false ); // phpcs:ignore WordPress.PHP.StrictInArray.FoundNonStrictFalse -- In case the value from $subscription['connections'] is a string.
```

Comment:
I've left the non-strict comparison for this call to `in_array`, in case the value from $subscription['connections'] is a string.